### ISSUE TITLE
TTML Return format of the Time values

### ISSUE THREAD

**Issue by joe-sciame-wm** [MediaArea/MediaInfoLib#2065]
For timed text files i.e. SMPTE-TT, IMSC, is it possible to check the format of the timecode values. TTML allows several timebase formats but we only allow specific ones.

- Make sure they are all the same. No mismatch
- Make sure they are either one of hh:mm:ss:ff or hh:mm:ss.sss.

As requested logging this ticket here for MediaInfo and will use in Media Conch

**Comment by JeromeMartinez** [comment]
We would list the time formats in a specific field (hidden by default in MediaInfo but usable in MediaConch) and you would be able to check with MediaConch with a rule "if field is HH:MM:SS:FF or field is "HH:MM:SS.mmm".

**Comment by joe-sciame-wm** [comment]
Yes, this would work. Let me know if we can get this into a build.

**Comment by joe-sciame-wm** [comment]
Hi Jerome,

When do you think we can get this into a build?

Thanks,
Joe

**Comment by JeromeMartinez** [comment]
Apologizes for the delay, I prioritize that for this week.
